DSharpPlus/Net/Models/ChannelEditModel.cs

```diff
@@ -12,6 +12,7 @@ public class ChannelEditModel : BaseEditModel
         public string Name { internal get; set; }
         public int? Position { internal get; set; }
         public string Topic { internal get; set; }
+        public bool Nsfw { internal get; set; }
```

Comment:
This should be `bool?`.

Reply:
I made this change on my fork. I don't know how to update the commit.